From 02d0408df1b2b1966abf274177985e78e56b697f Mon Sep 17 00:00:00 2001
From: Alexi Demers <alexidemers@gmail.com>
Date: Tue, 8 Nov 2022 20:54:59 -0500
Subject: [PATCH] remove spamming debug messages for ramp_delay

---
 drivers/regulator/core.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/regulator/core.c b/drivers/regulator/core.c
index 8bc62e949..57373e39e 100644
--- a/drivers/regulator/core.c
+++ b/drivers/regulator/core.c
@@ -2906,7 +2906,6 @@ static int _regulator_set_voltage_time(struct regulator_dev *rdev,
 		return rdev->constraints->settling_time_down;
 
 	if (ramp_delay == 0) {
-		rdev_dbg(rdev, "ramp_delay not set\n");
 		return 0;
 	}
 
-- 
2.34.1

