From da6f921b606cb5ead57e17c49b72378e8626c735 Mon Sep 17 00:00:00 2001
From: Alexi Demers <alexidemers@gmail.com>
Date: Fri, 6 Jan 2023 22:06:24 -0500
Subject: [PATCH] Add missing boot args for oragenpi-r1plus

---
 include/configs/rockchip-common.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/configs/rockchip-common.h b/include/configs/rockchip-common.h
index 1ee1c57..9d6da7b 100644
--- a/include/configs/rockchip-common.h
+++ b/include/configs/rockchip-common.h
@@ -136,7 +136,8 @@
 	"fi; \0"
 
 #define RKIMG_BOOTCOMMAND		\
-	"if load mmc 1:3 0x4000000 bankb; then \
+	"setenv bootargs \"${bootargs} earlycon=uart8250,mmio32,0xff130000 kpti=0 console=ttyFIQ0 rootwait coherent_pool=1m \"; \
+	if load mmc 1:3 0x4000000 bankb; then \
 		setenv bootargs \"${bootargs} root=${bootarg_rootb}\"; \
 	else \
 		setenv bootargs \"${bootargs} root=${bootarg_roota}\";\
